src/components/TopBar.js

```diff
@@ -307,6 +307,16 @@ const TopBar = Utils.connectStore(authStore, 'authState')(class TopBar extends C
             icon('virus', { size: 24, style: styles.nav.icon }),
             'COVID-19 Data & Tools'
           ]),
+          isBioDataCatalyst() && h(NavSection, {
+            href: Nav.getLink('privacy'),
+            onClick: () => this.hideNav(),
+            ...Utils.newTabLinkProps
+          }, ['Terra Privacy Policy']),
+          isBioDataCatalyst() && h(NavSection, {
```

Comment:
Is there a way to make it so I'm only checking `isBioDataCatalyst()` once and then create the two menu items?

Reply:
`h(Fragment, [])`

Reply:
thanks!